Adding new authorisation providers: dev environment

akuleshov7 opened the pull request:
### What's done:
- testing experiments with google, huawei and gitee
- dev configuration for oauth providers (no prod keys)
- frontend changes to support it


Mouse is over Github on the page:

<img width="1728" alt="image" src="https://github.com/saveourtool/save-cloud/assets/58667063/6645700e-1c6e-4401-83c9-bff0348e7848">

sanyavertolet reviewed (approved):
lgtm but Huawei logo needs to be aligned properly

akuleshov7 commented:
> lgtm but Huawei logo needs to be aligned properly as well as Gitee one

Easy to say, but yes, it’s my focus )

sanyavertolet commented:
> > lgtm but Huawei logo needs to be aligned properly as well as Gitee one
> 
> Easy to say, but yes, it’s my focus )

At least you could try `align-middle` on `div` that contains the logo. It will not fix sizes, but still

kgevorkyan commented on api-gateway/src/main/resources/application-dev.yml:
```diff
@@ -15,11 +15,16 @@ spring:
             authorization-uri: https://gitee.com/oauth/authorize
             token-uri: https://gitee.com/oauth/token
             user-info-uri: https://gitee.com/api/v5/user
+          # example: https://github.com/wearearima/spring-boot-dex/blob/master/src/main/resources/application.properties
```
if its necessary info, probably better to link the concrete commit hash, i.e permalink

```suggestion
          # example: https://github.com/wearearima/spring-boot-dex/blob/ad7dd7f501f8fc3946fc87184b4b8b705713ead9/src/main/resources/application.properties
```

kgevorkyan commented on api-gateway/src/main/resources/application-dev.yml:
```diff
@@ -52,6 +67,10 @@ spring:
           github:
+            # IMPORTANT! When you create a github application on the GITHUB you need to provide a callback API.
+            # If you will only provide a domain name: 'saveourtool.com' in CALLBACK URL field, everything will go well.
+            # Github will automatically
```
automatically ... ?

akuleshov7 commented:
> > > lgtm but Huawei logo needs to be aligned properly as well as Gitee one
> > 
> > 
> > Easy to say, but yes, it’s my focus )
> 
> At least you could try `align-middle` on `div` that contains the logo. It will not fix sizes, but still

I have changed everything, removed font awesome (as it is prohibited)
